tests/test_bayes.py

```diff
@@ -1,7 +1,92 @@
+import itertools
+from collections.abc import Sequence
+from typing import Any
+
 import pytest
 from hypothesis import given, note, strategies as st
 
-from hypofuzz.bayes import softmax
+from hypofuzz.bayes import (
+    BehaviorRates,
+    DistributeNodesTarget,
+    distribute_nodes,
+    softmax,
+)
+
+
+@given(
+    st.integers(1, 100),
+    st.lists(
+        st.builds(
+            DistributeNodesTarget,
+            nodeid=st.integers(0, 1000).map(lambda n: f"node{n}"),
+            rates=st.builds(
+                BehaviorRates,
+                per_input=st.floats(min_value=0, max_value=10**10, exclude_min=True),
+                per_second=st.floats(min_value=0, max_value=10**10, exclude_min=True),
+            ),
+            e_startup_time=st.floats(min_value=0, max_value=10**10, exclude_min=True),
```

Comment:
10e6 is probably a more realistic upper bound; ten billion seconds or 0.1ns don't really make sense here.